.github/actions/build-single-project/action.yml

```diff
@@ -54,7 +54,9 @@ runs:
         if [ ! -e "/usr/local/bin/ninja" ]; then
           ln -s "/usr/bin/ninja" "/usr/local/bin/ninja"
         fi
-
+    - name: "Install Cmake"
```

Comment:
Are we able to remove the ninja step above as well? It should be included in cmake.

Reply:
```
$ ~/Android/Sdk/cmake/3.22.1/bin/ninja --version
1.10.2
```

Reply:
let me try, i was going to ask you about it.

Reply:
hmm it passed but idk if it really tried to build or jus used the cache. i guess i need to change on of the WM elevant files and tr in another branch.

Reply:
i couldn't figure out what in workmanager requires ninja. mightve been inspection which is not included anymore.